src/components/Audioplayer/Segments/index.js

```diff
@@ -1,290 +1,56 @@
 import React, { Component, PropTypes } from 'react';
-import ReactDOM from 'react-dom';
-import { decrypt } from 'sjcl';
+import Helmet from 'react-helmet';
+
+import debug from 'helpers/debug';
```

Comment:
@sabeurthabti not an alias but prefilling the namespace for the debug lib. https://github.com/quran/quran.com-frontend/blob/master/src/helpers/debug.js